src/MCPServer/share/mysql_dev.sh

```diff
@@ -1,37 +1,77 @@
 #!/bin/bash
 
-databaseName=$1
-dbpassword=$2  # should be "" or "-p<password>", like in localDevSetup/recreateDB.sh
```

Comment:
Good catch.

On Tue, Dec 1, 2015 at 5:10 PM, Holly Becker <notifications@github.com>
wrote:

> In src/MCPServer/share/mysql_dev.sh
> <https://github.com/artefactual/archivematica/pull/336#discussion_r46365429>
> :
>
> > @@ -1,37 +1,77 @@
> >  #!/bin/bash
> >
> > -databaseName=$1
> > -dbpassword=$2  # should be "" or "-p<password>", like in localDevSetup/recreateDB.sh
>
> Old behavior was $dbpassword was -p <password> or "", so that it handled
> an empty password without requiring more input. This isn't preserved - -p
> always prompts, even when the password is empty, most notably when running
> the mysql_dev scripts.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/artefactual/archivematica/pull/336/files#r46365429>.
>



-- 
Jesús García Crespo,
Software Engineer, Artefactual Systems Inc.
http://www.artefactual.com | +1.604.527.2056